doc/source/reference/routines.linalg.rst

```diff
@@ -18,6 +18,18 @@ or specify the processor architecture.
 .. _OpenBLAS: https://www.openblas.net/
 .. _threadpoolctl: https://github.com/joblib/threadpoolctl
 
+The SciPy library also contains a ``linalg`` submodule, and there is overlap
+in the functionality provided by the SciPy and NumPy submodules.  SciPy
+contains functions not found in `numpy.linalg`, such as functions related to
+LU decomposition and the Schur decomposition, multiple ways of calculating the
+pseudoinverse, and matrix transcendentals such as the matrix logarithm.  Some
+functions that exist in both have augmented functionality in `scipy.linalg`.
+For example, `scipy.linalg.eig` can take a second matrix argument for solving
+generalized eigenvalue problems.  Some functions in NumPy, however, have more
+flexible broadcasting options.  For example, `numpy.linalg.solve` can handle
+"stacked" arrays, while `scipy.linalg.solve` accepts only a single square
+array as its first argument.
+
```

Comment:
Looks good. This text is specific in the examples but does not make a recommendation, which is what I think we want.